src/sentry/api/serializers/models/alert_rule.py

```diff
@@ -105,10 +108,11 @@ def get_attrs(self, item_list, user, **kwargs):
         for alert_rule in alert_rules.values():
             if alert_rule.owner_id:
                 owner_type = actor_type_to_string(alert_rule.owner.type)
-                if alert_rule.owner_id in resolved_actors[owner_type]:
-                    result[alert_rule][
-                        "owner"
-                    ] = f"{owner_type}:{resolved_actors[owner_type][alert_rule.owner_id]}"
+                if owner_type:
```

Comment:
I had to put this extra conditional because `actor_type_to_string` could return `None`.